core/src/test/java/io/questdb/test/griffin/DropStatementTest.java

```diff
@@ -0,0 +1,232 @@
+/*******************************************************************************
+ *     ___                  _   ____  ____
+ *    / _ \ _   _  ___  ___| |_|  _ \| __ )
+ *   | | | | | | |/ _ \/ __| __| | | |  _ \
+ *   | |_| | |_| |  __/\__ \ |_| |_| | |_) |
+ *    \__\_\\__,_|\___||___/\__|____/|____/
+ *
+ *  Copyright (c) 2014-2019 Appsicle
+ *  Copyright (c) 2019-2023 QuestDB
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ ******************************************************************************/
+
+package io.questdb.test.griffin;
+
+import io.questdb.cairo.CairoException;
+import io.questdb.cairo.TableToken;
+import io.questdb.cairo.TableWriter;
+import io.questdb.cairo.sql.RecordCursor;
+import io.questdb.cairo.sql.RecordCursorFactory;
+import io.questdb.griffin.CompiledQuery;
+import io.questdb.griffin.SqlException;
+import io.questdb.std.ObjHashSet;
+import io.questdb.test.AbstractGriffinTest;
+import io.questdb.test.tools.TestUtils;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class DropStatementTest extends AbstractGriffinTest {
+    /* **
+     * DROP can be followed by:
+     * - TABLE name [;]
+     * - TABLES name(,name)* [;]
+     * - DATABASE [;]
+     */
+
+    private final ObjHashSet<TableToken> tableBucket = new ObjHashSet<>();
+
```

Comment:
To save trouble in the future - could you change to double quotes around table names ?
Single quotes should be used only for string literals .

Reply:
yes, I have, it is misleading for me because I read the logs and all table names feature in single quotes. Then I have been using Python, where both single and double quotes have the same meaning, then to me a table name is a literal, so tend to use single quote. I will use double quotes for table names in SQL statements, and single quote in log "statements". I will try to keep sanity while doing it.